### PR TITLE
[DEV-6223] Add methods to dry up service classes

### PR DESCRIPTION
**Pull request by juliaBP**
https://benchprep.atlassian.net/browse/DEV-6223

### CODE REVIEW

File: lib/primer/base_service.rb [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
-      def set_running_command command
+      attr_reader :running_command, :port, :repo_name
+
+      def set_port(port)
+        @port = port
+      end
+
+      def set_repo_name(repo_name)
+        @repo_name = repo_name
+      end
```
Comment (nick-desteffen) [inline lib/primer/base_service.rb:12]: I don't see this used anywhere.

Comment (jbanass) [reply lib/primer/base_service.rb:12]: @nick-desteffen It appears we use it here:

https://github.com/watermelonexpress/benchprep-v2/pull/10745/files#diff-17b0bf160be262cf71f54b2c48d1469677b6c24817ab69370fa7d60b6091b68aR10

Comment (jbanass) [reply lib/primer/base_service.rb:12]: I think despite that we may be able to remove it entirely and just create a `getter`

https://github.com/watermelonexpress/benchprep-v2/pull/10745#discussion_r657088820

**Review by jbanass — COMMENTED** [review]
Just noting here that its good, but pending the convo in https://github.com/watermelonexpress/benchprep-v2/pull/10745#discussion_r657088820, we may be able to clean this up more.

**Comment by jbanass** [comment]
@juliaBP I'd appreciate your eyes on this

**Review by nick-desteffen — APPROVED** [review]

**Pull request merged**